### DISCUSSION TITLE
abstract class error

### DISCUSSION THREAD

**Discussion by simon-lake** [microsoft/vscode-discussions#882] [Extension Development QnA]
when i run debug i get the following error message: Abstract class 'ExcelWriter' with abstract methods instantiated (Ln 244)

my code appears to run normally with no known errors stopping the run. Is this something that i need to address or can i ignore it

**Comment by isidorn** [comment]
This discussion board is a place for vscode extension authors. For all other inqueries I suggest to head over to StackOverflow
